css/css-values/calc-rgb-percent-001.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Values Test: computed value of rgb() with calc(percentage)</title>
+
+  <!--
+
+  Original code is from Yves Lafon
+
+  Re: [css3-values] percentage in calc() and attr()
+  https://lists.w3.org/Archives/Public/www-style/2016May/0028.html
+
+  -->
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-values-4/#calc-serialize">
+
+  <script src="/resources/testharness.js"></script>
+
+  <script src="/resources/testharnessreport.js"></script>
+
+  <style>
+  div#target
+    {
+      background-color: rgb(calc(10% + 40%), calc(100% / 2), calc(25% * 2));
+      height: 100px;
+    }
+  </style>
+
+  <div id="target"></div>
+
+  <script>
+  test(function()
+    {
+      assert_equals(getComputedStyle(document.getElementById("target")).backgroundColor, "rgb(128, 128, 128)");
```

Comment:
This is relying on the current behavior of only allowing integers; 50% resolves to 127.5, which rounds up to 128 in computed style. UAs are allowed to support more depth, tho, so a 10-bit screen would correctly return `127.5` here.

I suggest swapping the percentages to resolve to a multiple of 20%, as those exactly hit integers in the 0-255 range. So perhaps `rgb(calc(5% + 15%), calc(40% / 2), calc(5% * 4))`, and then testing that it equals `rgb(51, 51, 51)`.